### DISCUSSION TITLE
JHEMCU GHF411 PRO F4 board compatability

### DISCUSSION THREAD

**Discussion by erreib** [iNavFlight/inav#7286] [Q&A]
Hello, I'm new to RC quadcopter building and am still a little overwhelmed by the amount of stuff involved, so I'm sorry if this is the wrong place to ask this. I'm pretty sure I know the answer to this already, but I wanted to confirm. I am planning on using JHEMCU GHF411 PRO F4 35A FC (http://www.jhemcu.com/e_productshow/?17-JHEMCU-GHF411AIO-F4-OSD-FLIGHT-CONTROLLER-BUILT-IN-30A-BL_S-2-4S-4IN1-ESC-FOR-TOOTHPICK-17.html) and wanted to see if there was any way to run iNav on this board? It comes with Betaflight installed but as I'm going for a quad with longer range, many of iNav's assist features are very appealing. As I understand it to get this working I would need to choose an existing target build of a different board with as close a design as possible to the one I have and learn how to remap the outputs from there. Thanks!

**Comment by JulioCesarMatias** [comment]
> Hello, I'm new to RC quadcopter building and am still a little overwhelmed by the amount of stuff involved, so I'm sorry if this is the wrong place to ask this. I'm pretty sure I know the answer to this already, but I wanted to confirm. I am planning on using JHEMCU GHF411 PRO F4 35A FC (http://www.jhemcu.com/e_productshow/?17-JHEMCU-GHF411AIO-F4-OSD-FLIGHT-CONTROLLER-BUILT-IN-30A-BL_S-2-4S-4IN1-ESC-FOR-TOOTHPICK-17.html) and wanted to see if there was any way to run iNav on this board? It comes with Betaflight installed but as I'm going for a quad with longer range, many of iNav's assist features are very appealing. As I understand it to get this working I would need to choose an existing target build of a different board with as close a design as possible to the one I have and learn how to remap the outputs from there. Thanks!

No, this FC is not iNav compatible.
